docs/dataset/formatting-your-datasets.md

```diff
@@ -90,14 +98,63 @@ These annotations are visible on both the Gallery view for groups of datapoints
 
 ### Structured Data
 
+Consider a `.csv` file containing ground truth data in the from of bounding boxes for an Object Detection problem.
+
+| locator                                                                       | label      | min_x     | max_x  | min_y | max_y   |
+|-------------------------------------------------------------------------------|------------|-----------|--------|-------|---------|
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | motorcycle | 270.77    | 621.61 | 44.59 |  254.18  |
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | car        | 538.03    | 636.85 | 8.86  | 101.93  |
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | trunk      | 313.02    | 553.98 | 12.01 | 99.84   |
+
+The first bounding box for the image is `(270.77, 44.59), (621.61,  254.18)`. To represent this within Kolena use the
+[`BoundingBox`](../reference/annotation.md#kolena.annotation.BoundingBox) annotation. This looks like:
+
+```python
+from kolena.annotation import BoundingBox
+bbox = BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61,  254.18))
+```
+When viewing a bounding box within python the format is
+`BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61, 254.18), width=350.84, height=209.59, area=73532.5556, aspect_ratio=1.67)`
+
+A single bounding box would be serialized as the following JSON string within a CSV:
+
+```
+{""top_left"": [270.77, 44.59], ""bottom_right"": [621.61, 254.18], ""width"": 350.84, ""height"": 209.59,
+ ""area"": 73532.5556, ""aspect_ratio"": 1.67, ""data_type"": ""ANNOTATION/BOUNDING_BOX""},
+```
+
+The above example has multiple objects within a single image, in order to ingest this you would need a single field
+containing the list of bounding boxes.
```

Comment:
I know the answer but this might make someone question why they can't use multiple fields for each bounding box.
Can we shorten this to  "The above example has multiple objects within a single image. This can be represented by a list of bounding boxes."

Reply:
Updated this